src/AutorestSwift/Models/Complex/Value.swift

```diff
@@ -74,7 +74,7 @@ class Value: Codable, LanguageShortcut {
 
     enum CodingKeys: String, CodingKey {
         case schema
-        case internalRequired = "required"
+        case required // = "required"
```

Comment:
We should probably just remove the comment if it isn't needed.

Reply:
Removed.